app/models/github_model.rb

```diff
@@ -49,7 +52,33 @@ def initialize(client, id_attributes, **options)
         attr[gh_attr.to_sym] = github_response(client, id_attributes.values.compact, options).send(gh_attr)
       end
 
-      remove_instance_variable("@response")
+      local_cached_attributes.each do |gh_attr|
```

Comment:
This takes the list of attributes defined in `local_cached_attributes` and defines the caching method for each one.